inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -149,7 +150,7 @@ DEFINE_uint32(nthreads, 0, infer_num_threads_message);
 DEFINE_string(nstreams, "", infer_num_streams_message);
 
 /// @brief Enforces bf16 execution with bfloat16 precision on systems having this capability
-DEFINE_bool(enforcebf16, false, enforce_bf16_message);
+DEFINE_string(enforcebf16, "True", enforce_bf16_message);
```

Comment:
It is works fine without this modification. You just need specify -enforcebf16=false. Please rollback this change and correct message. Otherwise behavior with flag without argument will be different in C++ and python version.

Reply:
Reverted